pyfr/nputil.py

```diff
@@ -35,7 +35,9 @@ def newfn(*args, **kwargs):
 
                 i, ix = 0, amix[0]
                 for j, jx in enumerate(amix[1:], start=1):
-                    if amfl[jx] - amfl[ix] >= tol:
+                    rdiff = ((amfl[jx] - amfl[ix]) /
```

Comment:
I think there might be a numpy function for this; see `np.allclose` and its arguments.